external/README.md

```diff
@@ -1,3 +1,27 @@
 # External Dependencies
 
-TODO
+## Cap'n Proto v0.5.2
+
+**Note**: Compiling on newer versions of linux with glibc >=2.14 can be
+problematic. A change to memcpy causes linking to the newer glibc version,
+resulting in binaries that cannot be used on older platforms. See details
+and a potential workaround here:
+http://www.win.tue.nl/~aeb/linux/misc/gcc-semibug.html
+
+Some compiler flags are required and can be specified as below with the
+`CXXFLAGS` environment variable.
+
+```
+curl -O https://capnproto.org/capnproto-c++-0.5.2.tar.gz
+tar zxf capnproto-c++-0.5.2.tar.gz
+cd capnproto-c++-0.5.2
+CXXFLAGS="-fPIC -std=c++11 -m64 -fvisibility=hidden -Wall -Wreturn-type -Wunused -Wno-unused-parameter" ./configure
+make -j3 check
```

Comment:
You can see exactly what I'm running here: https://github.com/rhyolight/nupic.devtools/blob/master/bin/install_nupic_core#L100-L169

The complete console output of running this script is here: https://gist.github.com/rhyolight/f23aa2415e40b8804bb2